### PR TITLE
Devise auth

### PR DESCRIPTION
**Pull request by floresv**
### Summary
Add and config devise_token_auth gem and update users table.

### Trello Card
https://trello.com/c/cCeZFekV/26-implementar-autenticaci%C3%B3n

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ end
 
 # Auth devise
 gem 'devise', '~> 4.4', '>= 4.4.3'
+gem 'devise_token_auth'
```
Comment (leamotta) [inline Gemfile]: remember to put the fixed version :)

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules.
+  devise :database_authenticatable, :registerable, :recoverable,
+         :rememberable, :trackable, :validatable, :confirmable
```
Comment (leamotta) [inline app/models/user.rb]: 🤔  you have almost the same code a few lines below

**Review by leamotta — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (floresv) [reply Gemfile]: Sorry, I forget it.

File: app/models/user.rb (hunk 2 above)
Comment (floresv) [reply app/models/user.rb]: Ok, thanks!

**Review by leamotta — APPROVED** [review]

**Pull request merged**